app/javascript/src/Users/utils/index.jsx

```diff
@@ -0,0 +1,44 @@
+// @flow
+
+import type { Feature, AdminSections } from 'Users/types'
+
+const FEATURE_NAMES: { [feature: Feature]: string } = {
+  portal: 'Developer Portal',
+  finance: 'Billing',
+  settings: 'Settings',
+  partners: 'Developer Accounts -- Applications',
+  monitoring: 'Analytics',
+  plans: 'Integration & Application Plans',
+  policy_registry: 'Policy Registry'
+}
+
+const FEATURES_GRANTING_SERVICE_ACCESS = ['partners', 'monitoring', 'plans']
+
+export function getFeatureName (feature: Feature): string {
+  if (feature in FEATURE_NAMES) {
+    return FEATURE_NAMES[feature]
+  }
+
+  throw new Error(`${feature} is not a known feature`)
+}
+
+export function canFeatureSetServicePermissions (features: AdminSections | Array<AdminSections>): boolean {
```

Comment:
Now here having `AdminSections` and ` Array<AdminSections>` suggest a multidimensional Array. Could be fixed with https://github.com/3scale/porta/pull/604/files#r272903513